### PR TITLE
Branch update add person command

### LINKED ISSUE #67: Refactor AddPerson Command to include his/her company
- As a user, I want to add a contact under a company I am associated with, so I can keep track of key business relationships.
- As a user, I want to delete a contact associated with a company, so I can maintain an up-to-date list of contacts.
- As a user, I want to edit the details of a contact associated with a company, so I can keep their information accurate.

### PR DESCRIPTION
**Pull request by geraldngjx**
Fixes #67 

Notable Changes:
- Update addPerson command to reflect new changes
- Add dummy company as sample data to add unassigned Person to Company (Temporary)
- Update existing test cases

### CODE REVIEW

File: src/main/java/connectify/logic/commands/AddPersonCommand.java [hunk 1]
```diff
@@ -33,21 +38,24 @@ public class AddPersonCommand extends Command {
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
             + PREFIX_TAG + "friends "
-            + PREFIX_TAG + "owesMoney";
+            + PREFIX_TAG + "owesMoney "
+            + PREFIX_COMPANY + "1";
```
Comment (ryantzr1) [inline src/main/java/connectify/logic/commands/AddPersonCommand.java:42]: Okay i see that we are assigning people to company based on company index. This works!

File: src/main/java/connectify/logic/commands/AddPersonCommand.java [hunk 2]
```diff
@@ -56,7 +64,22 @@ public CommandResult execute(Model model) throws CommandException {
...
 
+        List<Company> lastShownList = model.getFilteredCompanyList();
+
+        if (companyIndex.getZeroBased() >= lastShownList.size()) {
+            throw new CommandException(Messages.MESSAGE_INVALID_COMPANY_DISPLAYED_INDEX);
+        }
+
+        Company affiliatedCompanyToEdit = lastShownList.get(companyIndex.getZeroBased());
+
+        Company editedAffliatedCompany = affiliatedCompanyToEdit.addPersonToCompany(toAdd);
```
Comment (ryantzr1) [inline src/main/java/connectify/logic/commands/AddPersonCommand.java:75]: Good use of composition to show how Company contains people.

File: src/main/java/connectify/model/ModelManager.java [hunk 3]
```diff
@@ -42,6 +42,8 @@ public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyUserPrefs userPrefs
         logger.fine("Initializing with address book: " + addressBook + " and user prefs " + userPrefs);
 
         this.addressBook = new AddressBook(addressBook);
+        // Add a default company to the address book - To be implemented in future
```
Comment (ryantzr1) [inline src/main/java/connectify/model/ModelManager.java:45]: Let's create an issue for this or take note under our weekly agenda

**Review by ryantzr1 — APPROVED** [review]
LGTM!

File: src/main/java/connectify/model/ModelManager.java (hunk 3 above)
Comment (geraldngjx) [reply src/main/java/connectify/model/ModelManager.java:45]: Agreed. Let's create an issue for this under Milestone v1.3.

**Pull request merged**